<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViVolumeWidget</class>
 <widget class="QWidget" name="ViVolumeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>95</width>
    <height>105</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>1050</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="3">
    <widget class="ViDialer" name="volumeDial">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="volumeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>37</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ViButton" name="muteButton" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViButton</class>
   <extends>QWidget</extends>
   <header>vibutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViDialer</class>
   <extends>QDial</extends>
   <header>vidialer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
